webapp/app/js/directives/clonejobs.js

```diff
@@ -646,49 +649,6 @@ treeherder.directive('thCloneJobs', [
         }
     };
 
-    /**
-     * Reset the counts on a single resultset.  This should happen once per
-     * resultset when the platforms in a resultset have been updated
-     * (by loading initial or updated data).
-     *
-     */
-    var resetCounts = function(resultSet){
```

Comment:
this counting is all handled in the ``resultset_store`` from mauro's changes.  So this was wasted effort.